bin/licensetool

```diff
@@ -3,23 +3,35 @@
 const fs = require('fs');
 const path = require('path');
 const util = require('util');
+const { stringify } = require('csv-stringify/sync');
 const { strict: assert } = require('node:assert');
 
 const licenseChecker = require('license-checker-rseidelsohn');
 const spdxExpressionParse = require('spdx-expression-parse');
 const spdxLicenseList = require('spdx-license-list/full');
 
 function parseArgs() {
-  // TODO: Use node:util.parseArgs
   // Currently only a limited syntax is accepted: "format --config <path>"
```

Comment:
I think the second line of this comment is a little bit outdated, too.
You could just remove it as well.

Reply:
Thank you. I removed the comment.